src/components/editor/Editor.jsx

```diff
@@ -36,6 +38,14 @@ class Editor extends Component {
     }
   }
 
+  handleRdfShow = () => {
```

Comment:
I think we may be able to move the handleRdfShow and handleRdfClose functions to the RDFModal component, see larger comment below.

Reply:
we decided to leave these here, at least for now.